pkgs/development/compilers/llvm/common/lldb.nix

```
@@ -29,15 +31,22 @@ stdenv.mkDerivation (rec {
 
   outputs = [ "out" "lib" "dev" ];
 
-  sourceRoot = if lib.versionOlder release_version "13" then null else "source/${pname}";
+  sourceRoot =
+    if lib.versionOlder release_version "13" then null
+    else if ((lib.versions.major release_version) == "13") then "source/${pname}" else "${src.name}/${pname}";
```

`llvmPackages_13.lldb.src.name == "source"`.

```suggestion
    else "${src.name}/${pname}";
```